include/wallaby/tello.h

```diff
@@ -0,0 +1,23 @@
+
+#ifndef INCLUDE_WALLABY_TELLO_H
+#define INCLUDE_WALLABY_TELLO_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+char * find_tellos();
+
+int connect_tello(char * tello);
```

Comment:
Shouldn't this be const?

Reply:
Also, consider making this `tello_connect` to be consistent with rest of libwallaby